src/log/console_appender.cpp

```diff
@@ -83,41 +83,51 @@ namespace fc {
       }
    }
 
+   string fixed_size( size_t s, const string& str ) {
+      if( str.size() == s ) return str;
+      if( str.size() > s ) return str.substr( 0, s );
+      string tmp = str;
+      tmp.append( s - str.size(), ' ' );
+      return tmp;
+   }
+
    void console_appender::log( const log_message& m ) {
       //fc::string message = fc::format_string( m.get_format(), m.get_data() );
       //fc::variant lmsg(m);
 
       FILE* out = stream::std_error ? stderr : stdout;
 
       //fc::string fmt_str = fc::format_string( cfg.format, mutable_variant_object(m.get_context())( "message", message)  );
-      std::stringstream file_line;
-      file_line << m.get_context().get_file() <<":"<<m.get_context().get_line_number() <<" ";
-
-      ///////////////
-      std::stringstream line;
-      line << variant(m.get_context().get_timestamp()).as_string() << " ";
-      line << std::setw( 10 ) << std::left << m.get_context().get_thread_name().substr(0,9).c_str() <<" "<<std::setw(30)<< std::left <<file_line.str();
+      
+      std::string file_line = m.get_context().get_file();

```

Comment:
Are you meaning to set file_line and then reset it in the next line?

Reply:
No. I'll fix that.